src/pages/guides/getting_started/code-playground-workflow.md

```diff
@@ -0,0 +1,188 @@
+6. **Test Your Add-on**: Click the **Run Code** button to execute your code within the context of your add-on
+
+## Keyboard Shortcuts
+
+Use these keyboard shortcuts to work more efficiently:
+
+| Action                           | Windows/Linux                                                | macOS                                                       |
+| -------------------------------- | ------------------------------------------------------------ | ----------------------------------------------------------- |
+| **Save**                         | <kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>S</kbd>            | <kbd>Cmd</kbd> + <kbd>Shift</kbd> + <kbd>S</kbd>            |
+| **Run**                          | <kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>Return/Enter</kbd> | <kbd>Cmd</kbd> + <kbd>Shift</kbd> + <kbd>Return/Enter</kbd> |
+| **Reset**                        | <kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>X</kbd>            | <kbd>Cmd</kbd> + <kbd>Shift</kbd> + <kbd>X</kbd>            |
+| **Increase font size**           | <kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>Plus (+)</kbd>     | <kbd>Cmd</kbd> + <kbd>Shift</kbd> + <kbd>Plus (+)</kbd>     |
+| **Decrease font size**           | <kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>Minus (-)</kbd>    | <kbd>Cmd</kbd> + <kbd>Shift</kbd> + <kbd>Minus (-)</kbd>    |
+| **Switch between tabs**          | <kbd>Ctrl</kbd> + <kbd>1, 2, 3, 4</kbd>                      | <kbd>Cmd</kbd> + <kbd>1, 2, 3, 4</kbd>                      |
+| **View the typings suggestions** | <kbd>Ctrl</kbd> + <kbd>space</kbd>                           | <kbd>Cmd</kbd> + <kbd>space</kbd>                           |
+
+<InlineAlert slots="header,text1" variant="success"/>
+
+#### TIP
+
+Use the "**More**" button in the top right corner of the playground window to reference the available keyboard shortcuts, start a new session, link to documentation and more.
+
+## Debugging
+
+The Code Playground supports debugging your code with the browser's developer tools. In **Add-on mode**, you can use the full power of the debugger for the add-on's UI and logic (the `HTML` and `IframeJS` tabs). The `Document JS` tab and the code in the **Script mode** editor are running in a sandboxed environment with [limited capabilities](../../references/document-sandbox/web/index.md); the best option here is to log messages to the Console.
+
+<InlineAlert slots="text, text1, text2" variant="info"/>
+
+The Code Playground **prefixes the messages in the Console** with descriptive strings to help you distinguish them from other messages.
+
+- `[Playground: Add-on]` for Code Playground in Add-on mode.
+- `[Playground: Script]` for Code Playground in Script mode.
+- `[Add-on: <add-on-name>]` for add-ons from other sources (including the CLI).
```

Comment:
This one is for the CLI. We should move this to the local development section of the docs.